java-manta-client-unshaded/src/main/java/com/joyent/manta/exception/MantaErrorCode.java

```diff
@@ -63,30 +63,30 @@
      */
     AUTH_SCHEME_ERROR("AuthScheme"),
     /**
-     * @since 3.4.1
+     * @since 3.5.0
```

Comment:
I'm confused why this PR would change a pre-existing `@since`.  That's not really something that we can change retroactively.